### ISSUE TITLE
Fix dependencies

### ISSUE THREAD

**Issue by lensbart** [eddeee888/graphql-code-generator-plugins#218]
When using Yarn (latest) with plug ‘n’ play/zero-installs, some stricter rules on dependencies apply. Specifically, packages can’t import anything not listed in their `package.json`. In this case, it seems like `tslib` is missing.

A workaround is to put the following in `.yarnrc.yml`:

```yml
# .yarnrc.yml
packageExtensions:
  # awaiting fix: https://github.com/eddeee888/graphql-code-generator-plugins/issues/218
  '@eddeee888/gcg-typescript-resolver-files@*':
    dependencies:
      'tslib': '*'
```

**Comment by eddeee888** [comment]
Thanks for reporting this @lensbart !

Could you please try this alpha version please:

```
yarn add -D @eddeee888/gcg-typescript-resolver-files@pr219-run357-1
```

I have added `tslib` into package.json's deps for this version.

**Comment by lensbart** [comment]
```
bart@MacBook-Air-4 codebase % yarn add -D @eddeee888/gcg-typescript-resolver-files@pr219-run357-1
➤ YN0000: · Yarn 4.0.2
➤ YN0000: ┌ Resolution step
➤ YN0085: │ + @eddeee888/gcg-typescript-resolver-files@npm:0.0.0-pr219-run357-1-20231214095843
➤ YN0085: │ - @eddeee888/gcg-typescript-resolver-files@npm:0.7.1
➤ YN0000: └ Completed in 0s 351ms
➤ YN0000: ┌ Post-resolution validation
➤ YN0069: │ @eddeee888/gcg-typescript-resolver-files ➤ dependencies ➤ tslib: This rule seems redundant when applied on the original package; the extension may have been applied upstream.
➤ YN0000: └ Completed
➤ YN0000: ┌ Fetch step
➤ YN0013: │ A package was added to the project, and one was removed (- 0.02 KiB).
➤ YN0000: └ Completed in 2s 64ms
➤ YN0000: ┌ Link step
➤ YN0000: │ ESM support for PnP uses the experimental loader API and is therefore experimental
➤ YN0000: └ Completed in 0s 234ms
➤ YN0000: · Done with warnings in 3s 358ms
```

This seems to work indeed! (See the line marked `YN0069`)

**Comment by eddeee888** [comment]
I have released this fix in `@eddeee888/gcg-typescript-resolver-files@0.7.2`. 

Thanks for taking the time to collaborate with me 🙏
